src/main/java/org/snomed/snowstorm/rest/ConceptController.java

```diff
@@ -133,14 +137,16 @@ public ItemsPage<?> findConcepts(
 			@RequestParam(required = false) Boolean isNullEffectiveTime,
 			@RequestParam(required = false) Boolean isPublished,
 			@RequestParam(required = false) String statedEcl,
+			@RequestParam(required = false, defaultValue = "false") Boolean includeLeafFlag,
+			@RequestParam(defaultValue = "inferred") Relationship.CharacteristicType form,
```

Comment:
There is a conflict here between the parameters `ecl`, `statedEcl` and `form` (which can be inferred or stated).
To ensure consistency perhaps the `statedEcl` parameter should be deprecated and the stated form used consistently across parameters when an argument of `form` = stated is given.